website/source/intro/getting-started/build-image.html.md

```diff
@@ -113,6 +113,12 @@ With a properly validated template. It is time to build your first image. This
 is done by calling `packer build` with the template file. The output should look
 similar to below. Note that this process typically takes a few minutes.
 
+-&gt; **Note:** For the tutorial it is convenient to use the credentials in the
+command line. However, it is potentially insecure. [Learn how to set this
+securely](/docs/builders/amazon.html#specifying-amazon-credentials). For
+example, with a credentials profile, you can do this:
```

Comment:
> However, it is potentially insecure. Learn how to set this securely.

In general I like the note. But I hesitate to keep this wording since it can easily be interpreted as _"the best way is to use a AWS config file"_. And it might be better to reference our docs about AWS credentials lookup https://www.packer.io/docs/builders/amazon.html .

> credentials profile

This is called `AWS Config File` by Amazon.